skills/named-to-qualifier/SKILL.md

```
@@ -0,0 +1,167 @@
+#[Attribute(Attribute::TARGET_PARAMETER)]
+#[Qualifier]
+final class {ClassName}
+{
+}
+```
+
+## グループ化の指針
+
+関連するQualifierは同じディレクトリにまとめる：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)